README.md

```diff
@@ -21,12 +21,12 @@
 * **name** - a series of any number of labels (including 0) separated by label separators, e.g. `abc.eth`.
```

Comment:
I think this would be duplicating the same definition many times (the jump from label to name).
First we introduce the concept of a label, then a name.
Next we define a normalized name referring to ENSIP-15 directly.
We also mention that it contains non-empty labels, but that is already repeating what is described in ENSIP-15.
All other definitions refer to the previous "name" and "normalized name" definitions, which means we don't have to bring up how beautified/curable names are also made up of labels.
So I think the current way is ok.